Polish/updates vs2013

gonzalozawa opened the pull request:
@denishoctor I have done some work in here (fix the issues, polish and updates). Could you please assign it to someone for review? 
Summary of the changes:
- Fix Assess Example: This was not working it was a simple fix so it is committed and merged to master.
- Fix Author Example: there was a wrong reference to the author api and some minor issues that were making the whole example fail. Now both example (author item edit and author item list) work.
- Update the solution to Visual Studio 2013.
- Add reference to Newtonsoft.Json package. Now there is no need to reference the library when installing the SDK. The package is automatically downloaded and installed using NuGet if necessary.
- Fix reference to LearnositySDK project. Now there is no need to reference the LearnositySDK.dll when installing the SDK.
- Include Example/Views/Shared/LrnContainer.cshtml to the solution.
- Delete Dist folder. No needed anymore after adding Newtonsoft package and reference to LearnositySDK project.

denishoctor commented:
https://learnosity.atlassian.net/browse/LRN-4518 has been added to S15 for review. Good chuck of effort here so we don't want to rush it. Thanks Gonzalo.

gonzalozawa commented:
I have added one more change. Working in the .NET SDK Quickstart Guides for the docs I have realized that we do not have a proper handling of the credentials. So I have added them to the web.config file and created a static class to handle them.

jack-vo commented on Example/Views/JS/Author.cshtml:
```diff
@@ -9,6 +9,7 @@
 <script>
     document.getElementById('code').innerHTML = library.json.prettyPrint(@Html.Raw(ViewBag.JSON));
 </script>
+<script src="//authorapi.learnosity.com?v0.10"></script>
```
@gonzalozawa as explained in the ticket, we do not want to render our API inside the example as if our API's changes break the API we have to go back to fix the example. Please remove this <script>

jack-vo commented on Example/Views/JS/Author.cshtml:
```diff
@@ -9,6 +9,7 @@
+<script src="//authorapi.learnosity.com?v0.10"></script>
 <script>
     var initOptions = @Html.Raw(ViewBag.JSON);
     LearnosityAuthor.init(initOptions);
```
@gonzalozawa and in here please do not call `LearnosityAuthor.init(initOptions);`

gonzalozawa commented:
@jack-vo changes have been applied and Example project been removed. I have also added a new solution file (default and VS2010 version). Please let me know your comments.
